src/createAnimatedComponent/JSPropsUpdater.ts

```diff
@@ -0,0 +1,170 @@
+'use strict';
+import {
+  NativeEventEmitter,
+  NativeModules,
+  findNodeHandle,
+} from 'react-native';
+import {
+  nativeShouldBeMock,
```

Comment:
There's already a PR which eliminates `nativeShouldBeMock` (#5310), do you think we should use `!shouldBeUseWeb()` here or do it in a separate PR?

Reply:
I did it in this PR 🫡